lib/in/get-latest-from-registry.js

```diff
@@ -20,19 +22,23 @@ function getNpmInfo(packageName) {
 
             const latest = rawData['dist-tags'].latest;
             const next = rawData['dist-tags'].next;
-            const latestStableRelease = semver.satisfies(latest, '*') ?
-                latest :
-                semver.maxSatisfying(sortedVersions, '*');
+            const latestStableRelease = semver.satisfies(latest, '*')
+                ? latest
+                : semver.maxSatisfying(sortedVersions, '*');
+            const lastModified = rawData.time.modified;
```

Comment:
@lorenzleutgeb Since it is not further specified in https://docs.npmjs.com/cli/view what changes the modified property tracks exactly, I decided to go with the latestStableRelease timestamp as suggested to be on the safe side, thanks a lot!

Reply:
Yep, that's one of the reasons why I proposed to use date of the latest stable release as reference. One other reason is that consumers of packages do not profit from unreleased modifications.